renovate.json

```diff
@@ -1,3 +1,34 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "prConcurrentLimit": 5,
+  "extends": [
+    "config:js-app",
+    "group:monorepos",
+    "group:recommended",
+    "packages:eslint",
+    "packages:unitTest",
+    "packages:tslint"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["spur-"],
+      "groupName": "Spur packages",
+      "description": "All packages related to spur node framework"
+    },
+    {
+      "packagePatterns": ["prettier", "-prettier"],
+      "groupName": "Prettier packages",
+      "description": "All packages related to the prettier plugin"
+    },
+    {
+      "packagePatterns": ["eslint", "eslint-"],
+      "groupName": "ESLint packages",
+      "description": "All packages related to the eslint plugin"
+    },
+    {
+      "packagePatterns": ["jest", "@types/jest", "nock", "jest-"],
```

Comment:
Are you able to extend from Renovate's [built-in Jest + types group](https://docs.renovatebot.com/presets-group/#groupjestplustypes)? `group:jestPlusTypes`

Reply:
Hmm. I'll review this as a whole. I also recall Arnold mentioning that we have predefined config so I don't have to do this, but have not done the research yet. For now I am sticking to the common config we use.